api_essentials/rate_limiting.md

```diff
@@ -6,20 +6,34 @@ section_title: API Essentials
 
 ---
 
-Usage of Procore's API is subject to rate limits.
-By limiting the number of API requests that can be issued from a unique authentication token, we are able to offer a more reliable service by protecting our own system infrastructure from being negatively affected by abusive users or applications that are not operating in accordance with our terms and conditions.
+Usage of Procore's API is subject to rate limits. By limiting the number of API requests that can be issued from a unique authentication token, we are able to offer a more reliable service by protecting our own system infrastructure from being negatively affected by abusive users or applications that are not operating in accordance with our terms and conditions.
 
 ## Procore Rate Limits
 
-Procore API rate limits to 3,600 requests per hour.
-The rate limit resets every hour.
-There are three important response headers returned when making a request to the Procore API.
-
-| Header                 | Explanation                                                                     |
```

Comment:
@jimj-procoretech I completely understand why you are suggesting changes to tables.
To write tables in markdown means that each line of code could be super-long.

Still, this might be an indication that we should not use tables, but a different layout elements.

In fact [WWW Accessibility Rules](https://www.levelaccess.com/trouble-tables-brief-introduction/) remind us that the purpose of the `<table>` tag is to display tabular data, not to provide a specific layout for the HTML.

--- 

In this specific document, for instance, the text says:

> There are three important response headers returned when making a request to the Procore API.

What comes next, then, might be better specified as a **list** rather than a **table**, e.g.:

```markdown
- `X-Rate-Limit-Limit`: the total number of requests per 60 minute window
- `X-Rate-Limit-Remaining`: the number of requests you are allowed to make in the current 60 minute window
- `X-Rate-Limit-Reset`: the Unix timestamp for when the next window begins
```

or could even just be written in plain text in the same paragraph.

---

Using HTML inside a Markdown file opens a can of works to the maintanability and readability of this project.

- The same code goes from 5 lines to 23 lines of code
- When a new column is added to the table, maintainers will have to remember to edit both the `<thead>` and the `<tbody>` at once
- Markdown helpers such as `{% link … %}` are not available inside the HTML block, requiring maintainers to ad custom HTML and losing the benefits of link validations

---

In short, I agree with you that there are some tables in the current Markdown code that "look weird", with very long lines of code. My suggestion would be to identify them and to change them to a different layout (plain paragraphs or lists), while keeping Markdown tables for tabular data. What do you think?

Reply:
That sounds good. Can we do re-evaluation and fixing of the tables once we've cut over to the new system?

Reply:
@jimj-procoretech Yes we can! 

Would you like me to pull the remaining changes from this PR into a separate one, so we can apply those right away? I'm talking about the typos you find, the markdown mistakes, the extra blank lines at the end of some files…

Reply:
Sure sounds great!